recipe/meta.yaml

```diff
@@ -19,14 +19,19 @@ build:
 requirements:
   build:
     - python         # [win]
-    - toolchain
+    - cmake          # [win]
     - perl 5.22.2.1  # [not win]
-    - cmake  # [win]
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('fortran') }}  # [not win]
 
 test:
   requires:
     - python {{ os.environ.get('PY_VER', '') + '*' }}  # [win]
```

Comment:
Remove the 2 lines above

Reply:
You mean the compilers? I think those are necessary. How would you do it?

Reply:
Not the compilers. Line 28 and 29